src/tcgwars/logic/impl/gen4/RisingRivals.groovy

```diff
@@ -1937,19 +1929,33 @@ public enum RisingRivals implements LogicCardInfo {
               damage 30
               afterDamage {
                 targeted (defending) {
-                  bc "During ${opp.owner.getPlayerUsername(bg)}'s next turn, Energy can't be attached from their hand to the Defending ${defending}. (This effect can be removed by evolving or benching ${defending}.)"
+                  bc "During ${opp.owner.getPlayerUsername(bg)}'s next turn, ${opp.owner.getPlayerUsername(bg)} can't play any Pokémon from their hand to Evolve or Level Up the defending $defending. (This effect can be removed by evolving or benching ${defending}.)"
```

Comment:
Is this broadcast needed since a warning explaining the issue pops up when the affected player tries to play a relevant card?